velox/exec/tests/HashJoinTest.cpp

```diff
@@ -150,87 +212,113 @@ class HashJoinTest : public HiveConnectorTestBase {
     }
     return count;
   }
+
+  const int numDrivers_;
+
+  VectorFuzzer::Options fuzzerOpts_;
+  VectorFuzzer fuzzer_;
+
+  // The default left and right table types used for test.
+  RowTypePtr leftType_;
+  RowTypePtr rightType_;
```

Comment:
Looks like they are only used in ParallelJoin.outOfJoinKeyColumnOrder. Do we still want these to be member fields?